### PR TITLE
Dịch "batch-norm_vn" - Phần 8

### LINKED ISSUE #1386: Dịch "batch-norm_vn" - Phần 8
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_convolutional-modern/batch-norm_vn.md#L502-L550

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by tranyenthy**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #1386

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm **[hướng dẫn hoặc yêu cầu dịch hình](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [ ] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Với Reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

3. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 1]
```diff
@@ -564,7 +564,9 @@ Likely, its resonance as a focal point of these debates owes to its broad recogn
 Batch normalization has proven an indispensable method, applied in nearly all deployed image classifiers, earning the paper that introduced the technique tens of thousands of citations.
 -->
 
-*dịch đoạn phía trên*
+Chúng tôi lưu ý rằng *sự dịch chuyển hiệp biến nội bộ* không đáng bị chỉ trích hơn bất kỳ hàng ngàn yêu cầu mơ hồ tương tự được đưa ra mỗi năm trong tài liệu kỹ thuật về ML .
```
Comment (lkhphuc) [inline chapter_convolutional-modern/batch-norm_vn.md:567]:
```suggestion
Chúng tôi lưu ý rằng *sự dịch chuyển hiệp biến nội bộ* không đáng bị chỉ trích hơn bất kỳ khẳng định nào trong số hàng ngàn  các lập luận mơ hồ được đưa ra mỗi năm trong nhiều tài liệu kỹ thuật về học máy .
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 2]
```diff
@@ -564,7 +564,9 @@ Likely, its resonance as a focal point of these debates owes to its broad recogn
...
+Có khả năng, sự cộng hưởng của nó như là một tâm điểm của những cuộc tranh luận này nhờ vào khả năng nhận diện rộng rãi của nó đối với đối tượng mục tiêu.
```
Comment (lkhphuc) [inline chapter_convolutional-modern/batch-norm_vn.md:568]:
```suggestion
Có khả năng, nó cộng hưởng như là một tâm điểm của những cuộc tranh luận này nhờ vào sự phổ biến của nó trong nhóm khán giả mục tiêu. 
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 3]
```diff
@@ -564,7 +564,9 @@ Likely, its resonance as a focal point of these debates owes to its broad recogn
...
+Chuẩn hóa theo Batch đã chứng minh là một phương pháp không thể thiếu, được áp dụng trong gần như tất cả các phân loại hình ảnh được triển khai, kiếm được hàng chục ngàn trích dẫn cho bài báo giới thiệu kỹ thuật về nó.
```
Comment (lkhphuc) [inline chapter_convolutional-modern/batch-norm_vn.md:569]:
```suggestion
Chuẩn hóa theo Batch đã chứng minh là một phương pháp không thể thiếu, được áp dụng trong gần như tất cả các bộ phân loại hình ảnh được triển khai, và bài báo kỹ thuật giới thiệu nó đã có tới hàng chục ngàn trích dẫn.
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 4]
```diff
@@ -580,7 +582,10 @@ Batch normalization has proven an indispensable method, applied in nearly all de
 * Batch Normalization has many beneficial side effects, primarily that of regularization. On the other hand, the original motivation of reducing covariate shift seems not to be a valid explanation.
 -->
 
-*dịch đoạn phía trên*
+* Trong quá trình đào tạo mô hình, chuẩn hóa theo batch liên tục điều chỉnh đầu ra trung gian của mạng nơ-ron bằng cách sử dụng giá trị trung bình và độ lệch chuẩn của minibatch, để các giá trị của đầu ra trung gian trong mỗi lớp trong mạng lưới thần kinh ổn định hơn.
+* Các phương pháp chuẩn hóa theo batch cho các lớp được kết nối đầy đủ và các lớp chập có chút khác nhau.
```
Comment (lkhphuc) [inline chapter_convolutional-modern/batch-norm_vn.md:586]:
```suggestion
* Các phương pháp chuẩn hóa theo batch cho các tầng kết nối đầy đủ và các tầng tích chập có chút khác biệt.
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 5]
```diff
@@ -580,7 +582,10 @@ Batch normalization has proven an indispensable method, applied in nearly all de
...
+* Giống như một lớp dropout, các tầng chuẩn hóa theo batch có kết quả tính toán khác nhau trong chế độ đào tạo và chế độ dự đoán.
```
Comment (lkhphuc) [inline chapter_convolutional-modern/batch-norm_vn.md:587]:
```suggestion
* Giống như tầng dropout, các tầng chuẩn hóa theo batch có kết quả tính toán khác nhau trong chế độ đào tạo và chế độ dự đoán.
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 6]
```diff
@@ -602,7 +607,17 @@ Batch normalization has proven an indispensable method, applied in nearly all de
 7. Research ideas: think of other normalization transforms that you can apply? Can you apply the probability integral transform? How about a full rank covariance estimate?
 -->
 
-*dịch đoạn phía trên*
+1. Chúng ta có thể loại bỏ phép biến đổi được kết nối đầy đủ affine trước khi chuẩn hóa theo batch hoặc tham số sai lệch trong tính toán tích chập không?
```
Comment (lkhphuc) [inline chapter_convolutional-modern/batch-norm_vn.md:610]:
```suggestion
1. Chúng ta có thể loại bỏ phép biến đổi affine được kết nối đầy đủ trước khi chuẩn hóa theo batch hay tham số độ chệch trong phép tính tích chập không?
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 7]
```diff
@@ -602,7 +607,17 @@ Batch normalization has proven an indispensable method, applied in nearly all de
...
+    * Tìm một chuyển hóa tương đương áp dụng trước lớp được kết nối đầy đủ.
+    * Cải cách này có hiệu quả. Tại sao (không)?
```
Comment (lkhphuc) [inline chapter_convolutional-modern/batch-norm_vn.md:612]:
```suggestion
    * Cải tiến này có hiệu quả không và tại sao?
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 8]
```diff
@@ -602,7 +607,17 @@ Batch normalization has proven an indispensable method, applied in nearly all de
...
+2. So sánh tỷ lệ học tập cho LeNet khi có sử dụng và không sử dụng chuẩn hóa theo batch.
```
Comment (lkhphuc) [inline chapter_convolutional-modern/batch-norm_vn.md:613]:
```suggestion
2. So sánh tốc độ học của LeNet khi có sử dụng và không sử dụng chuẩn hóa theo batch.
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 9]
```diff
@@ -602,7 +607,17 @@ Batch normalization has proven an indispensable method, applied in nearly all de
...
+    * Phác họa sự giảm lỗi trong đào tạo và kiểm tra lỗi.
```
Comment (lkhphuc) [inline chapter_convolutional-modern/batch-norm_vn.md:614]:
```suggestion
    * Phác họa sự giảm lỗi đào tạo và lỗi kiểm tra.
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 10]
```diff
@@ -602,7 +607,17 @@ Batch normalization has proven an indispensable method, applied in nearly all de
...
+    * Còn về miền hội tụ thì sao? Bạn có thể tạo ra tỷ lệ học tập lớn như thế nào?
```
Comment (lkhphuc) [inline chapter_convolutional-modern/batch-norm_vn.md:615]:
```suggestion
    * Còn về miền hội tụ thì sao? Bạn có thể chọn tốc độ học lớn tới đâu?
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 11]
```diff
@@ -602,7 +607,17 @@ Batch normalization has proven an indispensable method, applied in nearly all de
...
+3. Chúng ta có cần Chuẩn hóa theo Batch trong mỗi lớp không? Thử nghiệm với nó thì sao?
```
Comment (lkhphuc) [inline chapter_convolutional-modern/batch-norm_vn.md:616]:
```suggestion
3. Chúng ta có cần chuẩn hóa theo batch trong mỗi lớp không? Hãy thử nghiệm điều này.
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 12]
```diff
@@ -602,7 +607,17 @@ Batch normalization has proven an indispensable method, applied in nearly all de
...
+4. Bạn có thể thay thế Dropout bằng Chuẩn hóa theo Batch không? Hành vi thay đổi như thế nào?
+5. Khắc phục các hệ số `beta` và `gamma` (thêm tham số `grad_req='null'` tại thời điểm xây dựng để tránh tính toán gradient), đồng thời quan sát và phân tích kết quả.
```
Comment (lkhphuc) [inline chapter_convolutional-modern/batch-norm_vn.md:618]:
```suggestion
5. Cố định các hệ số `beta` và `gamma` (thêm tham số `grad_req='null'` tại thời điểm xây dựng để không tính gradient), đồng thời quan sát và phân tích kết quả.
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 13]
```diff
@@ -602,7 +607,17 @@ Batch normalization has proven an indispensable method, applied in nearly all de
...
+6. Xem lại tài liệu về Gluon cho `BatchNorm` để xem các ứng dụng khác cho Chuẩn hóa theo Batch.
```
Comment (lkhphuc) [inline chapter_convolutional-modern/batch-norm_vn.md:619]:
```suggestion
6. Xem tài liệu của Gluon về `BatchNorm` để xem các ứng dụng khác cho Chuẩn hóa theo Batch.
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 14]
```diff
@@ -602,7 +607,17 @@ Batch normalization has proven an indispensable method, applied in nearly all de
...
+    * Tìm một chuyển hóa tương đương áp dụng trước lớp được kết nối đầy đủ.
+    * Cải tiến này có hiệu quả không và tại sao?
+2. So sánh tốc độ học của LeNet khi có sử dụng và không sử dụng chuẩn hóa theo batch.
+    * Phác họa sự giảm lỗi đào tạo và lỗi kiểm tra.
+    * Còn về miền hội tụ thì sao? Bạn có thể chọn tốc độ học lớn tới đâu?
+3. Chúng ta có cần chuẩn hóa theo batch trong mỗi lớp không? Hãy thử nghiệm điều này.
+4. Bạn có thể thay thế Dropout bằng Chuẩn hóa theo Batch không? Hành vi thay đổi như thế nào?
+5. Cố định các hệ số `beta` và `gamma` (thêm tham số `grad_req='null'` tại thời điểm xây dựng để không tính gradient), đồng thời quan sát và phân tích kết quả.
+6. Xem tài liệu của Gluon về `BatchNorm` để xem các ứng dụng khác cho Chuẩn hóa theo Batch.
+7. Ý tưởng nghiên cứu: nghĩ về các biến đổi chuẩn hóa khác mà bạn có thể áp dụng? Bạn có thể áp dụng biến đổi tích phân xác suất không? Còn về ước lượng hiệp phương sai ma trận hạng tối đa thì sao?
```
Comment (minhduc0711) [inline chapter_convolutional-modern/batch-norm_vn.md:620]: chỗ này còn từ `full rank` để mình đi hỏi 
```suggestion
7. Ý tưởng nghiên cứu: hãy nghĩ về các phép biến đổi chuẩn hóa khác mà bạn có thể áp dụng? Bạn có thể áp dụng phép biến đổi tích phân xác suất không? Còn ước lượng ma trận hiệp phương sai có hạng tối đa thì sao?
```
